src/controllers/recipe.controller.ts

```diff
@@ -13,127 +13,125 @@ interface Pagination {
   limit: number;
 }
 
-export async function createRecipe(
-  req: Request,
-  res: Response
-): Promise<Response> {
-  try {
-    const { title, description, preparing, ingredients, url } =
-      req.body as CreateRecipeBody;
-    if (!title || !description || !preparing || !ingredients || !url) {
-      return res
-        .status(400)
-        .json({ errorMessage: "Please enter all required data." });
-    }
+export class RecipeController {
+  public async createRecipe(req: Request, res: Response): Promise<Response> {
+    try {
+      const { title, description, preparing, ingredients, url } =
+        req.body as CreateRecipeBody;
+      if (!title || !description || !preparing || !ingredients || !url) {
+        return res
+          .status(400)
+          .json({ errorMessage: "Please enter all required data." });
+      }
 
-    const recipeData: Recipe = {
-      title,
-      description,
-      preparing,
-      ingredients,
-      userId: req.userId,
-      url,
-    };
+      const recipeData: Recipe = {
+        title,
+        description,
+        preparing,
+        ingredients,
+        userId: req.userId,
+        url,
+      };
 
-    const savedRecipe = await recipeService.createRecipe(recipeData);
-    return res.send(savedRecipe);
-  } catch (err) {
-    return res.status(500).send();
+      const savedRecipe = await recipeService.createRecipe(recipeData);
+      return res.send(savedRecipe);
+    } catch (err) {
+      return res.status(500).send();
+    }
   }
-}
 
-// reading recipes - if user is admin, then he sees all,
-// else user sees only recipes created by him
-// skip: number of items to skip, limit: how much to show
-// added filtering
-export async function getRecipeList(
-  req: Request,
-  res: Response
-): Promise<Response<RecipeDocument[]>> {
-  try {
-    const { userId, admin } = req;
-    const { skip, limit, name } = req.query;
-    let parsedName;
-    let recipes;
-    const parsedSkip = Number(skip);
-    const parsedLimit = Number(limit);
+  // reading recipes - if user is admin, then he sees all,
+  // else user sees only recipes created by him
+  // skip: number of items to skip, limit: how much to show
```

Comment:
If you're documenting method in details, use JSDoc format
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html